doc/tor.md

```diff
@@ -46,7 +46,6 @@ versions of Tor see [Section 3](#3-automatically-listen-on-tor).*
 
 	HiddenServiceDir /var/lib/tor/bitcoin-service/
 	HiddenServicePort 8333 127.0.0.1:8334
-	HiddenServicePort 18333 127.0.0.1:18334
```

Comment:
Agree re: testnet. if people want to run Tor testnet (or signet) nodes, they tend to be developers who know how to change around the ports here. It's just confusing as it is.